bitnami/contour/Chart.yaml

```diff
@@ -22,4 +22,4 @@ maintainers:
 name: contour
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/contour
-version: 12.0.1
+version: 12.0.2
```

Comment:
@brenix Can you please update your branch with the changes from the main branch? The current branch of contour is 12.0.2 not 12.0.1. So you should also update the version from 12.0.2 to 12.0.3.

Reply:
I have rebased and updated the chart version